var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -33,6 +33,9 @@ class PyBglibpy(PythonPackage):
     depends_on('py-bluepy', type='run')
     depends_on('py-libsonata', type='run')
 
+    # skip import test, because neurodamus-core requirement is not satisfied
```

Comment:
Can we make this more explicit: 
```suggestion
    # skip import test, because neurodamus-core requirement is not satisfied; 
    # bglibpy needs HOC_LIBRARY_PATH to function
```